codecov.yml

```diff
@@ -0,0 +1 @@
+comment: false
```

Comment:
If you're using sonarqube then you shouldn't need to use codecov too, since sonar's scans can include coverage information.